h/static/scripts/cross-frame-bridge.coffee

```diff
@@ -0,0 +1,109 @@
+class CrossFrameBridge
```

Comment:
I think I'd prefer it if this component were called just "Bridge". I sure hope we're not going to have multiple types of Bridge, and [longer names aren't always better](http://doc.cat-v.org/bell_labs/pikestyle).